.circleci/config.yml

```diff
@@ -165,6 +165,17 @@ jobs:
       - run: apt-get update -y && apt-get install -y libgflags-dev
       - run: TEST_TMPDIR=/dev/shm && make V=1 -j16 unity_test | .circleci/cat_ignore_eagain
 
+  build-linux-gcc-4-8:
+    docker:
+      - image: ubuntu-1604:201903-01
+    resource_class: xlarge
+    steps:
+      - checkout
+      - run: pyenv global 3.5.2
+      - run: sudo apt-get install gcc-4.8 g++-4.8 && sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.8 50 && sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 50
+      - run: sudo apt-get update -y && sudo apt-get install -y libgflags-dev
+      - run: CC=gcc-4.8 CXX=g++-4.8 V=1 SKIP_LINK=1 make -j32 all | .circleci/cat_ignore_eagain
```

Comment:
Please include this comment from the Travis configuration about why SKIP_LINK is used. It could take someone a while to reverse engineer that justification.

`Linking broken because libgflags compiled with newer ABI`